src/StimulusBundle/src/Form/Extension/FormTypeExtension.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Form\Extension;
+
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\Form\Extension\Core\Type\FormType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\UX\StimulusBundle\Dto\StimulusAttributes;
+use Twig\Environment;
+use Twig\Loader\ArrayLoader;
+
+class FormTypeExtension extends AbstractTypeExtension
+{
+    private StimulusAttributes $stimulusAttributes;
+
+    public static function getExtendedTypes(): iterable
+    {
+        return [FormType::class];
+    }
+
+    public function buildView(FormView $view, FormInterface $form, array $options): void
+    {
+        if (
```

Comment:
In this method you repeat two times the same code, this can be reduce a lot i think